documentai/snippets/process_document_sample.py

```diff
@@ -62,9 +62,21 @@ def process_document_sample(
     # Load binary data
     raw_document = documentai.RawDocument(content=image_content, mime_type=mime_type)
 
+    # Optional: Additional configurations for processing.
+    # For more information: https://cloud.google.com/document-ai/docs/reference/rest/v1/ProcessOptions
```

Comment:
```suggestion
    # For more information: https://cloud.google.com/document-ai/docs/reference/rest/v1/ProcessOptions
    # Optional: Additional configurations for processing.
```
Please check and add `Optional` comment just above the option field.